build.gradle

```diff
@@ -202,92 +202,144 @@ allprojects {
+    testImplementation group: 'org.hamcrest', name: 'hamcrest-library', version: hamcrestVersion
+    testImplementation group: 'org.jdom', name: 'jdom2', version: '2.0.6.1'
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junitVersion
+    testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junitVersion
     testImplementation group: 'org.mockito', name: 'mockito-inline', version: '3.11.1'
     testImplementation group: 'org.mockito', name: 'mockito-inline', version: '3.11.1'
+    testImplementation group: 'org.postgresql', name: 'postgresql', version: postgresqlVersion
     testImplementation group: 'org.powermock', name: 'powermock-api-mockito2', version: '2.0.9'
     testImplementation group: 'org.powermock', name: 'powermock-module-junit4', version: '2.0.9'
+    testImplementation group: 'org.projectlombok', name: 'lombok', version: lombokVersion
+    testImplementation("org.springframework.boot:spring-boot-starter-test") {
+      exclude group: 'junit', module: 'junit'
+      exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+    }
+    testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
+      exclude group: 'junit', module: 'junit'
+      exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+    }
+    testImplementation('org.springframework.cloud:spring-cloud-contract-wiremock:2.2.1.RELEASE') {
+      exclude group: 'commons-beanutils', module: 'commons-beanutils'
+    }
+    testImplementation('org.springframework.cloud:spring-cloud-starter-contract-stub-runner:2.1.0.RELEASE') {
+      exclude group: 'com.github.tomakehurst', module: 'wiremock-standalone'
+      exclude group: 'commons-beanutils', module: 'commons-beanutils'
+      exclude group: 'org.codehaus.groovy', module: 'groovy-xml'
+      exclude group: 'org.codehaus.groovy', module: 'groovy-json'
+      exclude group: 'org.codehaus.groovy', module: 'groovy'
+    }
+    testImplementation group: 'org.testcontainers', name: 'postgresql', version: testcontainers
+    testImplementation(platform('org.junit:junit-bom:5.8.2'))
     /*
-    Use actual implementation of xerces 2.12.2 instead of the one bundled with Java.  The version in openJDK contains
-    a bug in XMLSchemaValidator where cvc-complex-type.4 errors are reported twice.  The bug causes some XSD validation
-    tests to fail when running under openJDK 11.0.17 and above.  This dependency can be removed when the bug is fixed.
+      Use actual implementation of xerces 2.12.2 instead of the one bundled with Java.  The version in openJDK contains
+      a bug in XMLSchemaValidator where cvc-complex-type.4 errors are reported twice.  The bug causes some XSD validation
+      tests to fail when running under openJDK 11.0.17 and above.  This dependency can be removed when the bug is fixed.
     */
     testImplementation group: 'xerces', name: 'xercesImpl', version: '2.12.2'
```

Comment:
As above: Doesn't this mean this specification of xerces can be removed altogether i.e. we aren't using Java 11

Reply:
this needs to be used for now